ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -116,12 +115,8 @@ source "${SHARED_DIR}/govc.sh"
 
 DATACENTERS=("$GOVC_DATACENTER")
 # If testing a zonal install, there are multiple datacenters that will need to be cleaned up
-# vlanid between 1287 and 1302 will use profile:vsphere-multizone-2
-if [ ${vlanid} -ge 1287 ] && [ ${vlanid} -le 1302 ]; then
-  DATACENTERS=(
-    "IBMCloud"
-    "datacenter-2"
-  )
+if [[ "${vsphere_datacenters[*]}" -gt 0 ]]; then
```

Comment:
```suggestion
if [[ "${#vsphere_datacenters[@]}" -gt 0 ]]; then
```

Reply:
why add '#' here before the array? what's the meaning here?

Reply:
we are trying to get the length of `vsphere_datacenters` array to determinate if the cluster profile has multiple datacenters.

Reply:
`#` can be used to get the length of the array.